### ISSUE TITLE
Trying to get AwsomeBump built ? Win 7-64

### ISSUE THREAD

**Issue by gbic** [kmkolasinski/AwesomeBump#38]
Hi,
I'm 4 days in trying to get a nice texture for a rope in Blender and to be honest I'm past whits end trying to get this to build, total frustration doesn't come close to covering it.
I have downloaded and re-installed QT a number of times followed the instructions in your BuildingInstruction.pdf, (30 Apr 2015) I'm pretty sure I followed them correctly.  I must be missing something obvious this thing just won't build, so any help or suggestion would greatly appreciated.  really i just want to make a nice rope in Blender... :-(

OK heres what I have
Win 7-64 
![control panelall control panel itemssystem_2015-05-14_15-06-19](https://cloud.githubusercontent.com/assets/12426323/7625370/d8695eca-fa4a-11e4-8866-952fbefb6c9f.png)

QT Ver
![about qt creator_2015-05-14_15-08-52](https://cloud.githubusercontent.com/assets/12426323/7625424/a7d626c0-fa4b-11e4-8936-224363c99994.png)

MinGW Ver
![image](https://cloud.githubusercontent.com/assets/12426323/7625446/cbeeb28e-fa4b-11e4-93c8-4514e3972660.png)

Please let me know what else you need to know.

Cheers Willy

here is some more from the QT IDE
Under issues there is:
":-1: error: Qt Creator needs a compiler set up to build. Configure a compiler in the kit options."  
I have a compiler set ... at least I think I do?

under the General Messages:
QML module does not contain information about components contained in plugins.

Module path: D:/Qt/Qt5.4.1/5.4/msvc2013_64_opengl/qml
See "Using QML Modules with Plugins" in the documentation.

Automatic type dump of QML module failed.
Errors:
"D:\Qt\Qt5.4.1\5.4\msvc2013_64_opengl\bin\qmlplugindump.exe" crashed.
Arguments: --builtins

![awesomebump - qt creator_2015-05-14_15-37-46](https://cloud.githubusercontent.com/assets/12426323/7625702/4da8719a-fa4f-11e4-8846-7c06c5668c0e.png)

The play and build buttons are greyed out?

**Comment by Terrance8D** [comment]
AwesomeBump 3.14 binaries are available for Windows, is there a specific reason you need to compile from source? https://github.com/kmkolasinski/AwesomeBump/releases/tag/Winx32v3.1.4

**Comment by gbic** [comment]
Hi Thanks for the response.
I tried running just the binary (v3.1.4) and I get the following, I couldn't work out if there was a run time for QT and there didn't seem to be any other explanations as to how to fix. so I tried the build path.
![trying to get awsomebump built win 7-64 issue 38 kmkolasinskiawesomebump_2015-05-14_22-55-44](https://cloud.githubusercontent.com/assets/12426323/7630681/a84728e8-fa8c-11e4-8acd-ccff331a5195.png)
I'm in NZ so its just gone 23:00 I'll check back in the morning or tomorrow.  Thanks for your help.
Cheers Willy

**Comment by Terrance8D** [comment]
Well, I hate to tell you to reinstall Qt again, but you might just have to do that. I'm not on Windows anymore, so I'm not sure how much I'll be able to help.

**Comment by kmkolasinski** [comment]
Hi, 
1.) It seems you don't have installed Qt compiler only IDE. I don't know why but probably you just have to reinstall proper Qt package: try this link: https://www.qt.io/download/ and choose MinGW compiler toolbox during installation. It should work. 
2.) if you download the 3.14.exe file you also have to download full package which contains all necessary libraries: Download this package: https://github.com/kmkolasinski/AwesomeBump/releases/download/Winx32v3.0.4/AwesomeBumpV3.0.2Bin32x64Win7.zip
Then copy the AwesomeBump3.14.exe file to this folder. Since 3.0 I've started to upload only the "patch" file. Did you read description here:
https://github.com/kmkolasinski/AwesomeBump/releases/tag/Winx32v3.1.4

**Comment by gbic** [comment]
Hi guys,
many thanks.
first - i have it running so :+1: :+1: 2 thumbs up and thanks.  I feel a little sheepish as i didn't spot the full package above the 3.14 download.  It would be good if that listed under Downloads too.
Also might have saved me a few gigs of traffic but hey you got to try new things 

so to be clear I downloaded the Package above, unzipped it to a folder and then copied the latest 3.14 and its all go.

Now the fun start i guess, I have found a few bugs with the UI and I'll post these once I get a bit more familiar.  RE QT, I'm working on getting to grips with Blender so one challenge at a time, i might come back to QT one day.  Looks like a great tool, unfortunately my knowledge of how some of this works is lacking but I'm happy to put in the time.  I'll let you know how i get on with my rope ?

Many thanks again, Cheers Willy
